test_files/v2/channels.json

```diff
@@ -0,0 +1,30 @@
+{
+    "next": null,
+    "previous": null,
+    "results": [
+        {
+            "uuid": "09d23a05-47fe-11e4-bfe9-b8f6b119e9ab",
+            "name": "Android Phone",
+            "address": "+250788123123",
+            "country": "RW",
+            "device": {
+                "name": "Nexus 5X",
+                "power_level": 99,
+                "power_status": "STATUS_DISCHARGING",
+                "power_source": "BATTERY",
+                "network_type": "WIFI"
+            },
+            "last_seen": "2016-03-01T05:31:27.456000Z",
+            "created_on": "2014-06-23T09:34:12.866000Z"
```

Comment:
probably no one uses it, after all those who use API always like all details, And yes you can remove it until someone request it